frontend/www/js/omegaup/components/arena/Scoreboard.vue

```diff
@@ -105,6 +111,20 @@ export default {
         return '';
       }
     },
+    invitedClass: function(userIsInvited) {
+      if (userIsInvited == 1) {
```

Comment:
qué tal mejor

```suggestion
      return userIsInvited ? 'invited' : '';
```

(y recuerda que `userIsInvited` debe ser un booleano, no un entero, así que hay que asegurarnos que ese sea el caso).